### ISSUE TITLE
Contains method for empty collection

### ISSUE THREAD

**Issue by un-tone** [Starcounter/Starcounter.Linq#53]
Such query
```cs
DbLinq.Objects<Product>().Where(product => !names.Contains(product.Name));
```
is translated in an incorrect SQL:
```sql
SELECT P FROM "Product" P WHERE (NOT ())
```
when `names` collection is empty

**Comment by un-tone** [comment]
Fixed in versions [1.1](https://github.com/Starcounter/Starcounter.Linq/releases/tag/1.1.0), [2.1](https://github.com/Starcounter/Starcounter.Linq/releases/tag/2.1.0), [3.1](https://github.com/Starcounter/Starcounter.Linq/releases/tag/3.1.0)

**Comment by un-tone** [comment]
cc @PatrykSzwer
